swagger.yml

```diff
@@ -392,15 +392,16 @@ definitions:
       - GEOCODEDFAILED
       - OUT_OF_FSA
       - RATE_FAILED
+      - UNSUPPORTED_WEIGHT
       - GEOCODED
       - PICKUP_READY
       - CARRIER_PICKED_UP
       - SHIPMENT_CREATED
       - SHIPMENT_PROCESSING
+      - RECEIVED_AT_DESTINATION
       - OUT_FOR_DELIVERY
       - DELIVERED
       - FAILED
-      - CANCELLED
```

Comment:
`CANCELLED` is not used in this Hubble code to progress the order status (see below), so it's safe to say that we don't use it and that we can remove it:

```
export const ORDER_STATUS_TO_PROGRESS_MAP: { [key in OrderStatusType]?: number } = {
  [OrderStatus.ARCHIVED]: 0,
  [OrderStatus.CREATED]: 1,
  [OrderStatus.GEOCODEDFAILED]: 2,
  [OrderStatus.OUT_OF_FSA]: 3,
  [OrderStatus.RATE_FAILED]: 4,
  [OrderStatus.UNSUPPORTED_WEIGHT]: 5,
  [OrderStatus.GEOCODED]: 6,
  [OrderStatus.PICKUP_READY]: 7,
  [OrderStatus.CARRIER_PICKED_UP]: 8,
  [OrderStatus.SHIPMENT_CREATED]: 9,
  [OrderStatus.SHIPMENT_PROCESSING]: 10,
  [OrderStatus.RECEIVED_AT_DESTINATION]: 11,
  [OrderStatus.OUT_FOR_DELIVERY]: 12,
  [OrderStatus.FAILED]: 13,
  [OrderStatus.DELIVERED]: 14,
};
```